frontend/app-development/features/processEditor/ProcessEditor.test.tsx

```diff
@@ -145,7 +148,9 @@ describe('ProcessEditor', () => {
     });
 
     renderProcessEditor();
-    expect(consoleSpy).toHaveBeenCalledWith('SyncSuccess received');
+    await waitFor(() => {
+      expect(invalidateQueryByFileNameSpy).toHaveBeenCalledWith(syncSuccessMock.source.name);
```

Comment:
What about removing the spy and use the method directly from the instance itself?
 
```suggestion
      expect(invalidator.invalidateQueryByFileName).toHaveBeenCalledWith(syncSuccessMock.source.name);
```

Reply:
I reused some of the origin test code when it was testing logging success message. Agree that spy is unecessary to use in the updated test. Added a mock function instead.